iop/fs/bdmfs_fatfs/src/diskio.c

```diff
@@ -10,24 +10,24 @@
 #include <bdm.h>
 #include <cdvdman.h>
 
-#include "ff.h"     /* Obtains integer types */
-#include "diskio.h" /* Declarations of disk functions */
+#include "ff.h"			/* Obtains integer types */
+#include "diskio.h"		/* Declarations of disk functions */
+#include "fs_driver.h"  /* Declarations of fs driver functions */
 
-#include "fs_driver.h"
 
 /*-----------------------------------------------------------------------*/
 /* Get Drive Status                                                      */
 /*-----------------------------------------------------------------------*/
 
-DSTATUS disk_status(
-    BYTE pdrv /* Physical drive number to identify the drive */
+DSTATUS disk_status (
+	BYTE pdrv		/* Physical drive nmuber to identify the drive */
```

Comment:
typo here

Reply:
This typo belongs to the original code, I think it is fixed in the next release